modules/gdscript/tests/scripts/parser/features/unicode_identifiers.gd

```diff
@@ -1,4 +1,5 @@
 const π = PI
+@warning_ignore("confusable_identifier")
 var ㄥ = π
```

Comment:
This is now triggering `~~ WARNING at line 2: (CONFUSABLE_IDENTIFIER) The identifier "ㄥ" has misleading characters and might be confused with something else.` which is reasonable, but irrelevant for this test.